web/server/config/server_config.json

```
@@ -48,6 +48,12 @@
       "groups": [
         "adm", "cc-users"
       ]
+    },
+    "regex_groups": {
+      "enabled" : false,
+      "groups" : {
+        "admins_custom_group" : [ "whatever", "^root$", "^admin_", "_admin$" ]
```

It would be better to add real examples in this file because when a user starts a CodeChecker server in a new workspace this file will be created with these values. So for this reason I recommend to remove the `whatever` (nobody will use this matching) and the `^root$` (servers automatically create a root user to access the server's configuration via the clients / See the `CodeChecker server` command help /. And if a user is superuser it inherit all other permissions such as admin permission. )

Maybe it would be good to update the documentation too with these values.
```suggestion
        "admins_custom_group" : [ "admin", "^admin_", "_admin$" ]
```

---

I wanted to add as first entry a regex that wouldn't match the username used in the test, "regex_admin".